File: algorandsamples/mypythondemo/Indexer/SearchTxAddressBlock.py
import json
# requires Python SDK version 1.3 or higher
from algosdk.v2client import indexer

data = {
    "indexer_token": "",
    "indexer_address": "http://localhost:8980"
}

# instantiate indexer client
myindexer = indexer.IndexerClient(**data)
# gets tranactins for an account after a timestamp


# min/max rounds
data = {
    "address": "XIU7HGGAJ3QOTATPDSIIHPFVKMICXKHMOR2FJKHTVLII4FAOA3CYZQDLG4",
    "block": "7048877"
    }
response = myindexer.search_transactions_by_address(**data)
print("block: 7048877 = " +
      json.dumps(response, indent=2, sort_keys=True))

File: algorandsamples/myjsdemo/Indexer/SearchTxAddressBlock.js
//SearchTxAddressBlock.js
const algosdk = require('algosdk');
const indexer_token = "";
const indexer_server = "http://localhost";
const indexer_port = 8980;

// Instantiate the indexer client wrapper
let indexerClient = new algosdk.Indexer(indexer_token, indexer_server, indexer_port);

    
(async () => {
    let address = "XIU7HGGAJ3QOTATPDSIIHPFVKMICXKHMOR2FJKHTVLII4FAOA3CYZQDLG4";
    let round = 7048877;
    let response = await indexerClient.searchForTransactions()
        .address(address)
        .round(round).do();
    console.log("Information for Transaction search: " + JSON.stringify(response, undefined, 2));
    }
   
)().catch(e => {
    console.log(e);
    console.trace();
});